tests/bench/coremark/coremark_common.h

```diff
@@ -23,10 +23,16 @@ Original Author: Shay Gal-on
 /* Configuration: TOTAL_DATA_SIZE
         Define total size for data algorithms will operate on
 */
-#ifndef CONFIG_COREMARE_DATA_SIZE
 #ifndef TOTAL_DATA_SIZE
 #define TOTAL_DATA_SIZE 2 * 1000
 #endif
+
+#ifdef CONFIG_COREMARK_FORCE_VERY_QUICK
+#ifdef TOTAL_DATA_SIZE
+#undef TOTAL_DATA_SIZE
```

Comment:
Please try to think a way to avoid using undef.

Reply:
> Please try to think a way to avoid using undef.
> Say:
> #ifdef CONFIG_COREMARK_FORCE_VERY_QUICK
> #define TOTAL_DATA_SIZE...
> #endif
> #ifndef TOTAL_DATA_SIZE
> ...
> #endif

fixed